ffi/src/lib.rs

```diff
@@ -16,9 +16,19 @@ use delta_kernel::snapshot::Snapshot;
 use delta_kernel::{DeltaResult, Engine, Error, Table};
 use delta_kernel_ffi_macros::handle_descriptor;
 
+/// cbindgen:ignore
```

Comment:
```suggestion
// cbindgen doesn't parse our feature flags. So we tell it to ignore one of the declarations here to avoid errors due to it double including the handle module.
/// cbindgen:ignore
```